apm-agent-plugins/apm-asynchttpclient-plugin/src/test/java/co/elastic/apm/agent/asynchttpclient/AsyncHttpClientInstrumentationTest.java

```diff
@@ -49,11 +46,32 @@ public AsyncHttpClientInstrumentationTest(RequestExecutor requestExecutor) {
         this.requestExecutor = requestExecutor;
     }
 
+    public static AsyncHandler<Response> customAsyncHandler = new AsyncCompletionHandler<Response>() {
+        @Override
+        public State onStatusReceived(HttpResponseStatus responseStatus) {
+            assert(tracer.currentTransaction() != null);
```

Comment:
Use AssertJ, for consistency

Add static import for `org.assertj.core.api.Assertions.assertThat`

```suggestion
            assertThat(tracer.activeSpan()).isNotNull();
            assertThat(tracer.isExit()).isTrue();
```